### PR TITLE
LLM: enable previous models

### PR DESCRIPTION
**Pull request by WeiguangHan**
@hkvision Test and enable previous models

### CODE REVIEW

File: .github/workflows/llm_performance_tests.yml [hunk 1]
```diff
@@ -84,6 +84,42 @@ jobs:
...
+          current_date=$(date +%Y-%m-%d)
+          for file in *.csv; do
+              if [ -e "$file" ] && [[ "$file" == *"$current_date"* ]]; then
+                  filename="${file%.*}"
+                  extension="${file##*.}"
+
+                  new_filename="${filename}_test1.${extension}"
+                  mv "$file" "$new_filename"
+              fi
+          done
```
Comment (hkvision) [inline .github/workflows/llm_performance_tests.yml:96]: just use sed to change the file name in run.py

File: .github/workflows/llm_performance_tests.yml [hunk 2]
```diff
@@ -84,6 +84,42 @@ jobs:
...
+
+      - name: switch transformers version
```
Comment (hkvision) [inline .github/workflows/llm_performance_tests.yml:98]: may not need a different job, just upgrade transformers after you run once and re-run.

File: python/llm/test/benchmark/arc-perf-test2.yaml [hunk 3]
```diff
@@ -0,0 +1,14 @@
+# for the models which required the version of transformers 4.34.0
```
Comment (hkvision) [inline python/llm/test/benchmark/arc-perf-test2.yaml:1]: rename to arc-perf-transformers-434.yaml?

File: .github/workflows/llm_performance_tests.yml (hunk 1 above)
Comment (WeiguangHan) [reply .github/workflows/llm_performance_tests.yml]: ok

File: .github/workflows/llm_performance_tests.yml (hunk 2 above)
Comment (WeiguangHan) [reply .github/workflows/llm_performance_tests.yml]: ok

File: python/llm/test/benchmark/arc-perf-test2.yaml (hunk 3 above)
Comment (WeiguangHan) [reply python/llm/test/benchmark/arc-perf-test2.yaml]: ok

File: python/llm/test/benchmark/arc-perf-transformers-434.yaml [hunk 4]
```diff
@@ -0,0 +1,14 @@
+# for the models which required the version of transformers 4.34.0
```
Comment (hkvision) [inline python/llm/test/benchmark/arc-perf-transformers-434.yaml:1]: For the models that require transformers 4.34.0

**Review by hkvision — APPROVED** [review]

**Pull request merged**